prac_06/guitar.py

```diff
@@ -0,0 +1,19 @@
+class Guitar:
+    """Guitar information"""
+
+    def __init__(self, name="", year=0, cost=0):
+        self.name = name
+        self.year = year
+        self.cost = cost
+
+    def __str__(self):
+        return f"{self.name}, ({self.year}) : ${self.cost}"
+
+    def get_age(self, current_year=2021):
+        return current_year - self.year
+
+    def is_vintage(self, vintage=50):
```

Comment:
Same as before, could this be refactored as a global constant?  It might be better to have all of these things you can change (like current_year and vintage) in one spot at the top?